nixos/lib/testing/macos-host.nix

```
@@ -0,0 +1,17 @@
+{ config, lib, hostPkgs, ... }:
+
+{
+  config = lib.mkIf hostPkgs.stdenv.hostPlatform.isDarwin {
+    defaults = {
+      virtualisation.host.pkgs = hostPkgs;
+
+      # Mounting the whole nix store from the host is the default because
+      # this makes rebuilds of the VMs much faster and the VMs smaller.
```

```suggestion
      # On non-darwin, mounting the whole nix store from the host is the default because
      # this generally makes rebuilds of the VMs faster and the VMs smaller, while reducing
      # disk I/O.
```

The differences aren't immense, so I've toned it down a bit. Definitely reduces I/O though.